packages/api/src/schema/datasets/clinvar.js

```diff
@@ -0,0 +1,169 @@
+import {
+  GraphQLFloat,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql'
+
+import { mergeOverlappingRegions } from '../../utilities/region'
+import { lookupExonsByGeneId, lookupExonsByTranscriptId } from '../types/exon'
+import { VariantInterface } from '../types/variant'
+import { fetchAllSearchResults } from '../../utilities/elasticsearch'
+
+
+export const ClinvarVariantType = new GraphQLObjectType({
+  name: 'ClinvarVariant',
+  interfaces: [VariantInterface],
+  fields: {
+    // common variant fields
+    alt: { type: new GraphQLNonNull(GraphQLString) },
+    chrom: { type: new GraphQLNonNull(GraphQLString) },
+    pos: { type: new GraphQLNonNull(GraphQLInt) },
+    ref: { type: new GraphQLNonNull(GraphQLString) },
+    variantId: { type: new GraphQLNonNull(GraphQLString) },
+    xpos: { type: new GraphQLNonNull(GraphQLFloat) },
+    // ClinVar specific fields
+    alleleId: { type: GraphQLInt },
+    clinicalSignificance: { type: GraphQLString },
+    goldStars: { type: GraphQLInt },
+    majorConsequence: { type: GraphQLString },
+  },
+  isTypeOf: variantData => variantData.dataset === 'clinvar',
+})
+
+
+const rangeQueriesForRegions = (regions, padding = 75) => {
+  const paddedRegions = regions.map(region => ({
+    xstart: region.xstart - padding,
+    xstop: region.xstop + padding,
+  }))
+
+  const queryRegions = mergeOverlappingRegions(
+    paddedRegions.sort((r1, r2) => r1.xstart - r2.xstart)
+  )
+
+  return queryRegions.map(
+    ({ xstart, xstop }) => ({ range: { xpos: { gte: xstart, lte: xstop } } })
+  )
+}
+
+
+export const fetchClinvarVariantsInGene = async (geneId, ctx) => {
+  const geneExons = await lookupExonsByGeneId(ctx.database.gnomad, geneId)
+  const filteredExons = geneExons.filter(exon => exon.feature_type === 'CDS')
```

Comment:
Actually, this isn't quite consistent with the current browser. gnomAD includes 'UTR' and 'exon' regions as well. But elsewhere gnomAD beta uses only 'CDS' regions, so leaving it like this for now.